config/initializers/decidim.rb

```diff
@@ -125,3 +125,6 @@
 
 Rails.application.config.i18n.available_locales = Decidim.available_locales
 Rails.application.config.i18n.default_locale = Decidim.default_locale
+
```

Comment:
```suggestion
Rails.application.config.to_prepare do
  # Api tunnings
  Decidim::Api::Schema.max_complexity = 5000
  Decidim::Api::Schema.max_depth = 50
end
```
Can you add this? nothing related to the update but needed to make the awesome map work properly